addons/road-generator/nodes/road_container.gd

```diff
@@ -100,6 +109,9 @@ var _drag_init_transform # : Transform can't type as it needs to be nullable
 var _drag_source_rp: RoadPoint
 var _drag_target_rp: RoadPoint
 
+# Flag used internally during initial setup to avoid repeat generation
+var _is_ready := false
```

Comment:
I feel like there should be some way I can infer whether a node is officially ready or not (not to be confused with "entered tree" which is a signal that [happens earlier](https://x.com/gnumaru/status/1717880163847835799)), but having not found anything explicit, I found this solution worked the best for now.